.circleci/config.yml

```diff
@@ -155,7 +155,12 @@ jobs:
           root: "."
           paths:
             - "packages/**/dist"
+            - "packages/contracts-bedrock/cache"
+            - "packages/contracts-bedrock/artifacts"
             - "packages/contracts-bedrock/forge-artifacts"
+            - "packages/contracts-bedrock/tsconfig.tsbuildinfo"
+            - "packages/contracts-bedrock/tsconfig.build.tsbuildinfo"
+            - "packages/contracts-bedrock/scripts/go-ffi/go-ffi"
```

Comment:
Just to be safe, I've added everything I found in the `pnpm clean` command from the contracts package:

https://github.com/ethereum-optimism/optimism/blob/jm/ci-tweak/packages/contracts-bedrock/package.json#L32

Is there any cost to persisting more data here?

Reply:
theres no $ cost in terms of storage, it might take a bit longer to execute the persis_to_workspace step though

Reply:
OK, the step took 39 seconds on this commit, which is reasonable given the job historically has taken about 9 minutes, and this can hopefully shave 5+ minutes from `contracts-bedrock-checks`.